src/db/models/mint_config.rs

```diff
@@ -101,6 +101,15 @@ impl MintConfig {
             .load::<Self>(conn)?)
     }
 
+    /// Get mint config by id.
+    pub fn get_by_id(id: i32, conn: &Conn) -> Result<Option<Self>, Error> {
+        Ok(mint_configs::table
+            .filter(mint_configs::id.eq(id))
+            .limit(1)
```

Comment:
No need for this, `id` is guaranteed to be unique, and `.first()` will also limit to 1